pennylane_lightning/core/simulators/lightning_gpu/bindings/LGPUBindings_nb.hpp

```diff
@@ -78,14 +256,26 @@ void registerBackendSpecificObservables(nb::module_ &) {} // m
  * @param m Nanobind module
  */
 template <class StateVectorT>
-void registerBackendSpecificAlgorithms(nb::module_ &) {} // m
+void registerBackendSpecificAlgorithms([[maybe_unused]] nb::module_ &m) {} // m
```

Comment:
I don't this is necessary, but OK.

Reply:
You can remove // m, or whatever I commented beside the empty functions, when bringing the variable back to the function.